haskell/test/Test/Ligo/BaseDAO/Common.hs

```diff
@@ -42,6 +47,14 @@ import Test.Ligo.BaseDAO.Proposal.Config (ConfigDesc, fillConfig)
 
 type OriginateFn m = QuorumThreshold -> m DaoOriginateData
 
+type Seconds = Time Second
+
+doubleTime :: Time Second -> Time Second
+doubleTime c = timeMul 2 c
```

Comment:
Nitpick, but since we use `+:+` below we could also use `*:*` here

Reply:
Yes, changed.